### ISSUE TITLE
Horarius: Filtrer par groupe K/G/T

### ISSUE THREAD

**Issue by NewLunarFire** [antoineMoPa/glacial#8]
Presque toutes les périodes dans Horarius dépendent de vos groupes K (Projet), G (APP) ou T (Tutorat), sauf qu'on les voit tous dans Horarius. Ça serait cool de pouvoir rentrer nos groupes et de voir juste les périodes qui nous concernent. Sinon quand il y a 15 groupes de tutorat le lundi tu vois 15 périodes dans ton horaire quand en fait t'en as juste une qui te concerne.

**Comment by jbeaudoin11** [comment]
Oui, cest justement pour ca que cest rendu sous forme de addons qu'on load. Cest super simple de loader un script pour une autre fonctionalité

Perso je pense quon peu simplement faire un call sur le ical parse le fichier et refaire l'interface avec bootstrap ou autre

**Comment by antoineMoPa** [comment]
Refaire c'est une bonne idée, et on s'arrange pour que ça soit responsive au départ (mobile first) (Genre la vue pour une journée). Ensuite on assemble plusieurs jours ensembles pour refaire la vue par semaine. Et on ajoute les filtres.
